Assets/Scripts/Player/GhostModelMover.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+
+/// <summary>This Component causes the model to bob up and down (like a ghost!).</summary>
+public class GhostModelMover : MonoBehaviour
+{
+    [Tooltip("Number of bobs to make in one second")]
+    public float bobFrequency = 0.7f;
+    [Tooltip("Maximum vertical height of bobbing")]
+    public float bobHeight = 0.1f;
+    [Tooltip("Vertical offset to apply to bobbing")]
```

Comment:
should remove this, the `bobOffset` variable is unused and it makes for an unnecessary tooltip!